### DISCUSSION TITLE
Feldrandkurse

### DISCUSSION THREAD

**Discussion by MPNator** [Courseplay/Courseplay_FS22#978] [Q&A] [answered]
Hallo zusammen 
Ich habe auf der Erlengrat (Multiplayer, CP 7.0.1.0) meine Wiese in mehrere Abschnitte unterteilt und auch einen Feldrandkurs mit CP eingefahren .... dieser hat die Nummer CP-1 und wird mir auf der Karte auch angezeigt .... Leider kann ich den Feldrandkurs nicht wählen um damit einen Kurs zugenerieren. CP nimmt immer die komplette Wiese, die ihm dann doch tatsächlich etwas zu groß ist.

gibit es eine Möglichkeit den Feldrandkurs bei der Kursgenerierung anzuwählen?! ggf wäre es ja eine Überlegung diese "Randkurse" in den Generator als Dropdown einzufügen. Vielen Dank für eure spitzen Arbeit. 

LG

**Comment by Tensuko** [comment, marked as answer]
Feld innerhalb eines Feldes ist auch nicht vorgesehen.
Dafür ein weiteres setting wollten wir erst mal nicht machen.

**Comment by MPNator** [comment]
also hab ich im Prinzip keine Möglichkeit außer die ganze Bergwiese als einen Kurs zu berechnen?

**Reply by Tensuko** [reply]
Ja.

**Reply by Tensuko** [reply]
Bzw auch nicht ganz richtig. Du kannst mit dem Ingame Maltool versuchen einen Streifen Gras dazwischen zu malen, weil gemaltest Gras kein Feld ist.

**Reply by MPNator** [reply]
na dann versuch ich das mal ... ist schade dass es nicht anders geht bis dato aber mal abwarten was ihr noch zaubert :) Danke dir aufjedenfall für deine schnelle Hilfe
